packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -81,6 +81,29 @@ export class ICXOrderBook {
     )
   }
 
+  /**
+   * Create and submits a DFC HTLC transaction
+   *
+   * @param {HTLC} htlc
+   * @param {string} [htlc.offerTx] Transaction Id of the offer transaction for which the HTLC is
+   * @param {BigNumber} [htlc.amount] Amount in HTLC
+   * @param {string} [htlc.hash] Hash of seed used for the hash lock part
+   * @param {number} [htlc.timeout] Timeout (absolute in blocks) for expiration of HTLC in DFI blocks
+   * @param {UTXO[]} utxos Specific utxos to spend
+   * @param {string} [utxos.txid] transaction Id
+   * @param {number} [utxos.vout] The output number
+   * @return {Promise<ICXGenericResult>} Object indluding transaction id of the the transaction
+   */
+  async submitDFCHTLC (htlc: HTLC, utxos: UTXO[] = []): Promise<ICXGenericResult> {
```

Comment:
```suggestion
  async submitDFCHTLC (htlc: HTLC | ExtHTLC, utxos: UTXO[] = []): Promise<ICXGenericResult> {
```

Reply:
method overload
```ts
submitDFCHTLC (htlc: HTLC, utxos: UTXO[] = []): Promise<ICXGenericResult>
submitDFCHTLC (htlc: ExtHTLC, utxos: UTXO[] = []): Promise<ICXGenericResult>
async submitDFCHTLC (htlc: HTLC | ExtHTLC, utxos: UTXO[] = []): Promise<ICXGenericResult> {
``

Reply:
There is another method for submiting external HTLC `submitExtHTLC()`